newIDE/app/src/MainFrame/MainMenu.js

```diff
@@ -179,36 +179,6 @@ export const buildMainMenuDeclarativeTemplate = ({
         onClickSendEvent: 'main-menu-close',
         enabled: !!project,
       },
-      ...(!isMacLike() || !isApplicationTopLevelMenu
```

Comment:
all those options didn't feel like they belonged in the "File" menu anymore, most can be accessed from the homepage

Reply:
I think we might have some complaints about this, I remember VictrisGames working about having some options to match the usual file options that Windows show

Reply:
Now that all of these can be accessed from the homepage, this doesn't make a lot of sense to have them here I think. I'm ready to take the bet on this and revert if we're wrong